platform/android/java/lib/patches/com.google.android.vending.expansion.downloader.patch

```diff
@@ -175,7 +175,7 @@ index e4b1b0f1c..36cd6aacf 100644
 -import com.android.vending.expansion.downloader.R;
 +// -- GODOT start --
 +//import com.android.vending.expansion.downloader.R;
-+import com.godot.game.R;
++import org.godotengine.godot.R;
```

Comment:
For the reference, I don't know why those changes were needed in the first place. Might be worth reviewing whether they actually are, and if so see if we can contribute a change upstream to make these files package-agnostic.

Reply:
These changes are needed because we're not using the library correctly. It's intended to be used as an android library, actually in a similar way to what I'm doing in this PR. 
Directly adding the library source code into our own codebase is what's creating the need for the patch. I can look into cleanup that dependency later on.